trunk-recorder/systems/p25_parser.cc

```diff
@@ -562,13 +609,13 @@ std::vector<TrunkMessage> P25Parser::decode_tsbk(boost::dynamic_bitset<> &tsbk,
     message.message_type = DEREGISTRATION;
     message.source = si;
 
-    BOOST_LOG_TRIVIAL(debug) << "tsbk2f\tUnit Deregistration ACK\tSource ID: " << std::setw(7) << si << std::endl;
+    BOOST_LOG_TRIVIAL(debug) << "tsbk2f\tUnit Deregistration ACK\tSource ID: " << std::setw(7) << si;
   } else if (opcode == 0x30) { //
-    BOOST_LOG_TRIVIAL(debug) << "tsbk30";
+    BOOST_LOG_TRIVIAL(trace) << "tsbk30 TDMA SYNCHRONIZATION BROADCAST";
```

Comment:
trace or debug?

Reply:
I believe it should be ʼtraceʼ because it is a repetitive system message.
I think i changed it setting up a system where I saw it.